core/client/src/main/java/alluxio/client/block/RetryHandlingBlockWorkerClient.java

```diff
@@ -105,11 +110,7 @@ public void run() {
           }, Configuration.getInt(PropertyKey.USER_HEARTBEAT_INTERVAL_MS),
           Configuration.getInt(PropertyKey.USER_HEARTBEAT_INTERVAL_MS), TimeUnit.MILLISECONDS);
 
-      try {
-        sessionHeartbeat();
-      } catch (InterruptedException e) {
-        throw Throwables.propagate(e);
-      }
+      NUM_ACTIVE_SESSIONS.incrementAndGet();
```

Comment:
It is tracking the number of sessions active (not the number of heartbeats). It is to make sure that we close all the clients opened. Otherwise the session heartbeat will be there forever.